network/mq/pool.go

```diff
@@ -57,12 +65,11 @@ func (p *pool) Consume(ctx context.Context) error {
 		if p.mq.conn.IsClosed() {
 			log.Warn("MQ connection lost")
 
-			if p.retriesNumber == 0 {
-				log.Fatal("MQ connection closed")
-			}
+			for i := 0; i < len(retriesDelay); i++ {
```

Comment:
for simplicity maybe implement each 15 or 30 sec retry? should cover most use cases